_docs/installing/windows.md

```diff
@@ -10,8 +10,8 @@ order: 2
 
 MusicBot can be installed on Windows 7 through 11, though it requires installing some programs on your computer first.
 
-1. Install [Python 3.7](https://www.python.org/ftp/python/3.7.0/python-3.7.0.exe).
-2. During the setup, tick `Install launcher for all users (recommended)` and `Add Python 3.7 to PATH` when prompted.
+1. Install [Python 3.10](https://www.python.org/ftp/python/3.10.1/python-3.10.1-amd64.exe).
```

Comment:
Change this to 3.8 so we have the same behaviour on all platforms, even if 3.10 is newer